apps/explorer/src/app/routes/blocks/home/index.tsx

```diff
@@ -15,8 +15,8 @@ const Blocks = () => {
   return (
     <>
       <section>
-        <h1>Blocks</h1>
-        <button onClick={() => refetch()}>Refresh to see latest blocks</button>
+        <h1 className="route-header">Blocks</h1>
```

Comment:
Lets try and embrace tailwind a bit more and just make a <RouteHeader /> component. Added benefit to getting auto completion rather than having to remember/hunt down a className.

```suggestion
        <RouteHeader>Blocks</RouteHeader>
```